.env.example

```
@@ -1,4 +1,5 @@
 API_URL='http://192.168.1.3:3333'
 ALGOLIA_APP_ID=GC7K0ETHXB
 ALGOLIA_ADMIN_KEY=8ea4ffa0588206bb0c0751294097e875
-ALGOLIA_INDEX_NAME=searchable_data_staging
\ No newline at end of file
+ALGOLIA_INDEX_NAME_TECHNOLOGY=searchable_data_staging
+ALGOLIA_INDEX_NAME_SERVICE=searchable_service_staging
```

```suggestion
ALGOLIA_INDEX_NAME_TECHNOLOGY=searchable_technology_staging
ALGOLIA_INDEX_NAME_SERVICE=searchable_service_staging
```
Anyway, it's better define this variables like `APP_ENV:staging` and `ALGOLIA_PREFIX:searchable` and generate the index name dynamically from "prefix + entity name (like `technology` or `service`) + ambient sufix 🤔